lib/mix/tasks/smuggle.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Mix.Tasks.Smuggle do
+  @moduledoc ~S"""
+  ConfigSmuggler provides the `mix smuggle` task, which encodes
+  `config.exs`-style files into JSON-formatted key/value pairs.
+
+  Usage: `mix smuggle encode <filename.exs>`
+  """
+  use Mix.Task
+
+  @impl true
+  @shortdoc "Encodes a config.exs-style file into JSON keys and values"
+  def run(["encode", filename]) do
+    with {:ok, encoded_config_map} <- ConfigSmuggler.encode_file(filename) do
+      IO.puts("{")
+
+      encoded_config_map
+      |> Enum.map(fn {k,v} -> "  #{inspect(k)}: #{inspect(v)}" end)
```

Comment:
Don't we want to wrap these values in quotes?

Reply:
inspect/1 does that

Reply:
Right, carry on